components/card/card.css

```diff
@@ -1 +1,119 @@
-.card {}
+.card {
+  border-radius: 4px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-light);
+  display: flex;
+  flex-direction: column;
+  width: calc(var(--space-sm) * 21.875);
+  margin: var(--space-xl) 0;
+}
+
+.card-content {
+  display: flex;
```

Comment:
you can use the utility class here to have better reusability of the code.

Reply:
Yes, will be done as part of refactor